presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergDistributedQueries.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.testing.assertions.Assert;
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static com.facebook.presto.SystemSessionProperties.ITERATIVE_OPTIMIZER_TIMEOUT;
+import static com.facebook.presto.SystemSessionProperties.OPTIMIZER_USE_HISTOGRAMS;
+import static com.facebook.presto.common.type.VarcharType.VARCHAR;
+import static com.facebook.presto.testing.MaterializedResult.resultBuilder;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.IntStream.range;
+import static org.testng.Assert.assertTrue;
+
+@Test(singleThreaded = true)
+public abstract class TestIcebergDistributedQueries
+        extends AbstractTestDistributedQueries
+{
+    private final CatalogType catalogType;
+    private final Map<String, String> extraConnectorProperties;
+
+    protected TestIcebergDistributedQueries(CatalogType catalogType, Map<String, String> extraConnectorProperties)
+    {
+        this.catalogType = requireNonNull(catalogType, "catalogType is null");
+        this.extraConnectorProperties = requireNonNull(extraConnectorProperties, "extraConnectorProperties is null");
+    }
+
+    protected TestIcebergDistributedQueries(CatalogType catalogType)
+    {
+        this(catalogType, ImmutableMap.of());
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner() throws Exception
+    {
+        return IcebergQueryRunner.createIcebergQueryRunner(ImmutableMap.of(), catalogType, extraConnectorProperties);
+    }
+
+    @Override
+    protected boolean supportsNotNullColumns()
+    {
+        return false;
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+    }
+
+    @Override
+    public void testRenameColumn()
```

Comment:
I understand the code was previously like this, but could you add comments where necessary explaining that the Iceberg connector doesn't yet support these features?

Reply:
Sure! After recheck the tests here, I found some of them have been supported by Iceberg connector, so I remove these empty override methods directly. And in the remaining methods, comments have been added where necessary to indicate that the corresponding functionality is not supported by Iceberg connector or differs from the parent method being overridden. Please take a look when available, thanks!